config/prod/iatlas-runner.yaml

```diff
@@ -0,0 +1,16 @@
+template_path: gitlab-runner.yaml
+stack_name: iatlas-gitlab-runner-prod
+dependencies:
+  - common/iatlasvpc.yaml
+  - prod/iatlas-db.yaml
+  - prod/iatlas-kms.yaml
+stack_tags:
+  Department: "CompOnc"
+  Project: "iAtlas"
+  OwnerEmail: "andrew.lamb@sagebase.org"
+parameters:
+  InstanceType: "t2.nano"
+  VpcName: iastlasvpc
+  VpcSubnet: PrivateSubnet
+  AMIId: ami-0810a318c4b1243c5 #AWS Linux from https://ami.sageit.org
+  Environment: staging
```

Comment:
file is in config/prod so shouldn't environment be `prod`?

Reply:
Yes, yes it should be...